packages/mrm-task-lintstaged/index.js

```diff
@@ -32,9 +33,21 @@ function task(config) {
 
 		// Prettier
 		if (pkg.get('devDependencies.prettier') && !pkg.get('devDependencies.eslint-plugin-prettier')) {
+			const prettierVersion = semver.coerce(pkg.get('devDependencies.prettier'));
 			const script = pkg.getScript('format');
 			const exts = getExtsFromCommand(script);
-			newRules.push([extsToGlob(exts, prettierExtensions), 'prettier --write']);
+			let suggestedExtensions = prettierExtensions;
+			if (semver.satisfies(prettierVersion, '1.4.0 - 1.5.0')) {
+				// CSS was added in 1.4
+				suggestedExtensions = '.js,css';
```

Comment:
Shouldn't it be `.js,.css`?

Reply:
Not according to tests. It’s a weird format anyways. I was expecting it to be a glob or an array.